MiniSpace.Services.Reactions/src/MiniSpace.Services.Reactions.Application/Commands/CreateReaction.cs

```diff
@@ -0,0 +1,30 @@
+using System.Data;
+using System.Threading.Tasks.Dataflow;
+using Convey.CQRS.Commands;
+using MiniSpace.Services.Reactions.Core.Entities;
+
+namespace MiniSpace.Services.Reactions.Application.Commands
+{
+    public class CreateReaction : ICommand
+    {
+        public Guid ReactionId {get;}
+        public Guid StudentId { get; }
+        public string ReactionType { get; } 
+        
+        public Guid ContentId {get;}
+
+        // Is the reaction related to event or post?
```

Comment:
The comment is not necessary. It should be easy to understand what this field stands for